creme/activities/tests/test_forms.py

```diff
@@ -24,6 +23,8 @@
 from .base import _ActivitiesTestCase
 
 
+# from django.core.exceptions import ValidationError
+# from creme.creme_core.tests.forms.base import FieldTestCase
```

Comment:
It would be cool to re-read your code if your IDE make this kind of crappy moves...